packages/frontend/src/redux/slices/multiplier/index.js

```diff
@@ -2,7 +2,7 @@ import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 import merge from 'lodash.merge';
```

Comment:
`slices/multiplier/index.js`

The same as for `performBuyOrSellUSN.js`. I consider we can delete it from the state. Need to double check usage of these files.
Just a notice, we can do it later.

Reply:
@NotEternal 
Kindly create a task with this suggestion.